prefect_dbt/cloud/credentials.py

```diff
@@ -24,9 +24,36 @@ class DbtCloudCredentials:
         async with dbt_cloud_credentials.get_administrative_client() as client:
             client.trigger_job_run(job_id=1)
         ```
+
+        Load saved dbt Cloud credentials within a flow:
+        ```python
+        from prefect import flow
+
+        from prefect_dbt.cloud import DbtCloudCredentials
+        from prefect_dbt.cloud.jobs import trigger_job_run
+
+
+        @flow
+        def trigger_job_run_flow():
+            credentials = DbtCloudCredentials.load("my-dbt-credentials")
+
+            trigger_job_run(dbt_cloud_credentials=credentials, job_id=1)
+
+
+        trigger_job_run_flow()
+        ```
     """
 
-    api_key: str
+    _block_type_name = "dbt Cloud Credentials"
+    _logo_url = "https://asset.brandfetch.io/idofJOT4bu/idxrwTdDC-.svg"
+    _code_example = """/
```

Comment:
Maybe use of `from textwrap import dedent`?

Reply:
Yep, the `Block` base class handles that for us: https://github.com/PrefectHQ/orion/blob/main/src/prefect/blocks/core.py#L301-L303